### DISCUSSION TITLE
linguist-vendored doesn't work

### DISCUSSION THREAD

**Discussion by IgorHoholko** [github-linguist/linguist#5541] [Q&A] [answered]
Hello! 
Added my 3rd/ subfolder to **.gitattributes** but extra stats are still there :(
Checked 10 times for any mistakes. Looks okay. 

Thanks.

Repo:
https://github.com/IgorHoholko/DVC-Manager/tree/IgorHoholko-patch-1

**Comment by IgorHoholko** [comment]
Maybe it doesn't support folders with leading numbers in name :/

**Comment by lildude** [comment, marked as answer]
> Checked 10 times for any mistakes. Looks okay.

You've not made a mistake but you have [missed that `*` matches anything _except_ a slash](https://www.git-scm.com/docs/gitignore) (also discussed [here](https://github.com/github/linguist/discussions/5175#discussioncomment-317382)) which means it doesn't recurse so will only apply to files within the `dvc_listener/3rd/` directory and not to any of the sub-directories in that directory.

Replace `*` with `**` in either of your current overrides and remove the other (you don't need both if you're only trying to exclude files in `dvc_listener/3rd/`) and you're all set.

**Reply by IgorHoholko** [reply]
Thank you. 
That is true. One more thing to do is just wait a bit more to allow a worker to complete the analysis job :)
